go-controller/pkg/types/const.go

```diff
@@ -52,9 +52,26 @@ const (
 	V4NodeLocalNATSubnetNextHop    = "169.254.0.1"
 	V4NodeLocalDistributedGWPortIP = "169.254.0.2"
 
+	V4MasqueradeSubnet = "169.254.169.0/30"
+	V4HostMasqueradeIP = "169.254.169.2"
+	V6HostMasqueradeIP = "fd69::2"
+	V4OVNMasqueradeIP  = "169.254.169.1"
+	V6OVNMasqueradeIP  = "fd69::1"
+
 	// OpenFlow and Networking constants
 	RouteAdvertisementICMPType    = 134
 	NeighborAdvertisementICMPType = 136
 
 	OvnACLLoggingMeter = "acl-logging"
+
+	// LoadBalancer External Names
```

Comment:
it should be good to briefly explain what represent each category: cluster, worker, gateway